compose/ui/ui-test-junit4/src/desktopTest/kotlin/androidx/compose/ui/test/BasicTestTest.kt

```diff
@@ -111,4 +116,25 @@ class BasicTestTest {
         text = "2"
         rule.onNodeWithTag("text").assertTextEquals("2")
     }
+
+    @Test
+    fun testCaptureToImage() {
+        val color = Color.Green
+        rule.setContent {
+            Box(Modifier.testTag("box").size(20.dp).background(color))
+        }
+
+        val screenshot = rule.onNodeWithTag("box").captureToImage()
```

Comment:
If we introduce non-junit API, this test shouldn't use Junit

Reply:
I'm using junit intentionally. Didn't we have a goal to replicate Android's test framework API?

Reply:
> have a goal to replicate Android's test framework API?

If Android has it in ui-test as well, we can do that. To unblock commonization of ui-test, and to avoid dependency differences on user side.

Let's then move `SkikoTestOwner` into `ui-test`, and when we proceed to getting rid of junit, we could mark it `internal` safely 
in the future (because `runSkikoComposeUiTest` is experimental and we can do breaking cross-module changes).

Reply:
I don't understand.

Android's documented (and I presume primary) [testing API](https://developer.android.com/jetpack/compose/testing) is based on junit4, and that's what I'm working on replicating.

In the non-junit API, we already have `captureToImage`. What was missing, and what this PR adds, is the ability to use `captureToImage` with `ComposeTestRule` (a junit API).

Reply:
> What was missing, and what this PR adds, is the ability to use captureToImage with ComposeTestRule (a junit API).

Ahh. I mistakenly thought opposite, because we introduce new API in `ui-test`.

Reply:
> Let's then move `SkikoTestOwner` into `ui-test`

Ahh, I see what you mean. The reason I put `SkikoTestOwner` in `ui` is that the corresponding Android class (`ViewRootForTest`) is in `ui`. But I guess that's not important.